styles/globals.css

```diff
@@ -45,6 +45,22 @@ h2 {
   inset: 0;
 }
 
+.youtube-banner {
```

Comment:
Have you considered doing this with tailwind in the component itself? Was it difficult? We try to stick to one styling approach (Tailwind) if possible.

Reply:
My bad, was not sure if tailwind accepts custom heights and bg-color but found it in documentation, so now all styles for YT banner is in tailwind